Update README.md (#287)
SmartContractSecurity/SWC-registry: README.md

# This registry is no longer actively maintained

This registry has not been significantly updated since 2020, and is no longer actively maintained. While some maintenance may be done
to enhance or clarify the status of the current content, and pointers will be added to new work that is maintained such as the 
[EEA EthTrust Security Levels Specification](https://entethalliance.org/specs/ethtrust-sl/), new SWCs are no longer being added,
and readers should check external sources to clarify the relevance of existing content.

> 

Author: ```suggestion

All the work in here was incorporated into the **[EEA EthTrust Security Levels specification v1](https://entethalliance.org/specs/ethtrust-sl)**. 

The [EEA EthTrust Security Levels](https://entethalliance.org/groups/ethtrust) project is actively maintained. 
The **[Editor's draft for a new version](https://entethalliance.github.io/eta-registry/security-levels-spec.html)** is publicly available,
is updated roughly every two weeks, and a formal release is expected to be published in Q4 2023 as version 2 of the specification.

```

- Define a common language for describing security issues in smart contract systems' architecture, design, or code.
- Serve as a way to train and increase performance for smart contract security analysis tools.

## New SWC entries are not added since 2020

### Scope of Weaknesses

> SWCs aree concerned with weaknesses that can be identified within Solidity code of a smart contract.

Author: ```suggestion
SWCs are concerned with weaknesses that can be identified within Solidity code of a smart contract.
```

> Weaknesses in 'smart contract adjacent' code are not covered. 

Author: ```suggestion
Weaknesses in 'smart contract adjacent' code are not covered by this registry of Weaknesses. 
```

> Weaknesses in 'smart contract adjacent' code are not be included. 

Author: ```suggestion
Weaknesses in 'smart contract adjacent' code are not covered. 
```
